src/algebra/group/with_one.lean

```diff
@@ -126,10 +133,19 @@ end with_one
 
 namespace with_zero
 
+-- `to_additive` fails to generate some meta info around eqn lemmas, so `lift` doesn't workf
```

Comment:
```suggestion
-- `to_additive` fails to generate some meta info around eqn lemmas, so `lift` doesn't work
```
Would you mind moving these two lines of comments to the `can_lift` instance for `with_one`? Because that's where people might make a mental note "Hey, TODO: refactor this to use `to_additive`".